cernopendata/modules/fixtures/data/docs/cms-guide-troubleshooting/cms-guide-troubleshooting.md

```diff
@@ -1,34 +1,77 @@
 
-- In some versions of VirtualBox, the CMS Open Data VM does not open correctly. This has happened for example for VirtulBox 5.0.32, but a more recent version is OK.
+### Virtual machine
 
-- xml files do not download correctly by clicking on "Download" e.g. `http://opendata.cern.ch/record/560/files/BuildFile.xml` in [record 560](/record/560)
-Note that all files can be downloaded directly from the CMS Open Data VM terminal with wget e.g.
-```
-wget http://opendata.cern.ch/record/560/files/BuildFile.xml
-```
-
-- After the Windows update in June 2017, VirtualBox will not start. The problem has been fixed by the VirtualBox team and downloading the most recent version (5.1.22) from [the VirtualBox website](https://www.virtualbox.org/wiki/Downloads) solves the problem.
+**I have trouble trouble installing VirtualBox**
+- Read the FAQs in [the VM installation guide](/docs/cms-2011-virtual-machines-how-to-install#issue). If it still fails, please contact your local system administrator.
 
+**What is the root password for CMS Open Data VM?**
+- The root password for CMS Open Data VM is `password`.
 
-**I have trouble installing Virtualbox**
-> read the FAQs. If it still fails, please contact your local system administrator
+**The CMS Open Data VM does not open correctly**
+- In some versions of VirtualBox, it has happened that the CMS Open Data VM does not open correctly. This was the case for example for VirtulBox 5.0.32, but more recent versions from [the VirtualBox website](https://www.virtualbox.org/wiki/Downloads) have been tested and are working properly. Note that opening the CMS Open Data VM first time takes some time.
 
 **When/after installing CERN VM, I get a message that my VM uses too much memory**
-> reduce the memory allocated to the virtualbox
+- Reduce the memory allocated to VirtualBox by clicking on System in the VirtualBox graphical user interface and adjust the base memory with the sliding bar.
 
-**When reading AOD data, I get write access warnings from eospublic on every file**
-> this was a temporary 'feature' of a change in the eos software which has meanwhile been fixed. It does not affect the results.
+#### File download
 
-**When reading AOD data, I get fatal access error messages from eospublic on specific files**
-> According to the eos-admin team, disk access problems to eospublic may occur occasionally and are automatically corrected within a few hours. If the access problem to a particular file lasts longer than about a day send a mail to [eos-admins@cern.ch](mailto://eos-admins@cern.ch), providing the file name and a log of the error message.
+**xml files do not download correctly by clicking on "Download"** 
+- Note that all files can be downloaded directly from the CMS Open Data VM terminal with wget e.g. you download the file -`http://opendata.cern.ch/record/560/files/BuildFile.xml` in [record 560](/record/560) with
+`wget http://opendata.cern.ch/record/560/files/BuildFile.xml`
```

Comment:
Maybe change to `$ wget http://opendata.cern.ch/record/560/files/BuildFile.xml`?